lib/imagec/docker.go

```diff
@@ -136,8 +138,8 @@ func LearnAuthURL(op trace.Operation, options Options) (*url.URL, error) {
 
 	// Private registry returned the manifest directly as auth option is optional.
 	// https://github.com/docker/distribution/blob/master/docs/configuration.md#auth
-	if err == nil && options.Registry != DefaultDockerURL && fetcher.IsStatusOK() {
-		op.Debugf("%s does not support OAuth", url)
+	if err == nil && options.Registry != registry.DefaultV2Registry.Host && fetcher.IsStatusOK() {
+		log.Debugf("%s does not support OAuth", url)
```

Comment:
This seems like it should be using `op` given it's available. Same for the other new log message below.